apiserver/policyservice/policyimpl.go

```diff
@@ -25,10 +29,9 @@ import (
 	"github.com/crunchydata/postgres-operator/kubeapi"
 	"github.com/crunchydata/postgres-operator/util"
 	log "github.com/sirupsen/logrus"
-	"k8s.io/api/apps/v1"
+	v1 "k8s.io/api/apps/v1"
```

Comment:
I don't think we need the `v1` alias here as we already get it for free!

Reply:
For some reason my linter keeps adding `v1` to the front...

Reply:
Looks like this is expected behavior with `goimports`: https://github.com/golang/go/issues/30051#issuecomment-461913682

Reply:
Leaving as is since this `goimports` is working as designed.